client/connection/manager.go

```diff
@@ -62,9 +62,17 @@ func (manager *connectionManager) Connect(consumerID, providerID identity.Identi
 		}
 	}()
 
+	err = manager.startConnection(consumerID, providerID)
+	if err == utils.ErrRequestCancelled {
+		return ErrConnectionCancelled
```

Comment:
Do we really need separate error in manager if utils.ErrRequestCancelled is the only error which indicates cancelation ?

Reply:
Yes - we specify more concrete custom error message, which we use in our api response body.

Reply:
I was wondering, maybe this mapping can be done in api level - if manager.Connect returns `ErrRequestCancelled`, than we return custom error with response body we want.

Reply:
@tadovas pushed new commit of what I've meant.